index.js

```diff
@@ -281,7 +281,17 @@ function redisStore(args) {
       if (err) {
         return cb && cb(err);
       }
-      conn.del(key, handleResponse(conn, cb));
+
+      if (Object.prototype.toString.call(key) === '[object Array]') {
```

Comment:
I don't know how well supported `Array.isArray()` is in browsers. I used the code from the polyfill on MDN:  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray

I'll change it to `Array.isArray()`, if you'd like.